index.Rmd

```diff
@@ -2,18 +2,30 @@
 title: "Performance of Univariate Kernel Density Estimation methods in TensorFlow"
 subtitle: "Bachelor Thesis"
 author: ["Marc Steiner"]
-supervisor: ["Jonas Eschle", "Nicola Serra"]
+supervisor: ["Jonas Eschle", "Prof. Dr. Nicola Serra"]
 institute: ["University of Zurich"]
 keywords: ["Kernel Density Estimation", "Python", "TensorFlow", "Zfit"]
 site: "bookdown::bookdown_site"
-bibliography: ["citations/papers.bib"]
+bibliography: ["citations/papers.json"]
 link-citations: True
-csl: "citation-styles/aip.csl"
+csl: "citation-styles/aps-with-supscript.csl"
 ---
 
+# Acknowledgements {-}
+
+I am grateful to Prof. Dr. Nicola Serra, for giving me the opportunity to complete my bachelor thesis under his domain.
+
+I would also like to thank my supervisor Jonas Eschle, which advised me a lot during the research, implementation and writing. He provided both technical and emotional guidance and helped me fulfill my potential.
+
+Finally, I would like to thank my girlfriend Christa Schläppi, who unconditionally supported me through every challenge in the last year and acted as a sounding board for all the intelligent as well as not so intelligent ideas I had.
+
+\newpage
+
+
 # Abstract {-}
 
-Multiple implementations of a one-dimensional Kernel Density Estimation in TensorFlow/Zfit are proposed. Starting from the basic algorithm, several optimizations from recent papers are introduced and combined to ameliorate the efficiency of the algorithm. By comparing its accuracy and efficiency to implementations in pure Python it is shown as competitive and useful in real world applications.
+Multiple implementations of a one-dimensional Kernel Density Estimation based on TensorFlow and Zfit are proposed. Starting from the basic algorithm, several optimizations from recent papers are introduced and combined to ameliorate the efficiency of the algorithm. By comparing its accuracy and efficiency to implementations in pure Python it is shown as competitive and useful in real world applications. The newly proposed KDE implementation achieves state-of-the-art accuracy as well as efficiency for large one-dimensional data ($n \geq 10^8$) and may also provide runtime benefits for cases where the probability density estimate needs to be computed once but evaluated repeatedly.
```

Comment:
```suggestion
Kernel Density Estimation is a non-parametric density estimation and often used in statistical inference as done in High Energy Physics using frameworks like zfit. Multiple implementations of univariate Kernel Density Estimation based on TensorFlow, a just-in-time compiled mathematical Python library for CPU and GPU, and zfit are proposed. Starting from the exact algorithm, several optimizations from recent papers are introduced. These include linear binning, Fourier Transformed Kernels, Improved Sheather-Jones and Hofmeyr with specialized kernels. Their accuracy and efficiency is compared to existing implementations in Python and shown to be competitive. The newly proposed Kernel Density Estimation implementation achieves state-of-the-art accuracy as well as efficiency, especially for large number of kernels ($n \geq 10^8$).
```

Please feel free to accept the suggestion if you like it or change is in case you prefer. I've reworded a little and moved around in order to include